backend/django/core/templates/projects/update/umbrella.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+ {% block page_title %}Update Project{% endblock %}
+ {% load static %}
+ {% load render_bundle from webpack_loader %}
+
+ {% block content %}
+ <div class="row">
+   <div class="col-md-8 col-md-offset-2">
+     <div class="card">
+       <div class="cardface">
+         {{ form.media.css }}
+         <form action="." method="post" enctype="multipart/form-data">
+           {% csrf_token %}
+           <h1>Add Project To Folder</h1>
+           <h3>Description</h3>
+           <p>To help organize your labeling projects, you can assign projects to folders to organize your project list.</p>
+           <ul class="list-group">
+             <li class="list-group-item"><strong>Admins</strong> are able to update the project's folder to one they have created.</li>
+           </ul>
+           <h3>Instructions</h3>
+           <p>Select a folder from the dropdown field below to add your project. If you want to add the project to a folder that is not on the list, contact the folder's owner.</p>
```

Comment:
I'd also add "or type a new folder name in the text box to create a new folder and add this project to it." 

That way users know that they can create new folders in this update page.